auto_nag/team_managers.py

```diff
@@ -0,0 +1,32 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import json
+
+from auto_nag.people import People
+
+DEFAULT_PATH = "./auto_nag/scripts/configs/team_managers.json"
+
+
+class TeamManagers:
+    def __init__(self):
+        self._load_team_managers(DEFAULT_PATH)
+
+    def _load_team_managers(self, filepath):
+        peapole = People.get_instance()
```

Comment:
nit: typo

Reply:
Done in 57b46c3c937504c988dddd0a52f138ff5c9a93b8